### PR TITLE
feat: bump assets controllers

### PR DESCRIPTION
**Pull request by sahar-fehri**
## **Description**

Bumps assets-controllers version

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by sahar-fehri** [comment]
@metamaskbot update-policies

**Comment by sahar-fehri** [comment]
@metamaskbot update-policies

**Review by mcmire — DISMISSED** [review]
We probably need to bump `approval-controller` as I notice we're a bit behind on that, but we can do that in another PR. This PR looks good.

File: package.json [hunk 1]
```diff
@@ -249,7 +249,7 @@
     "@metamask/address-book-controller": "^3.0.0",
     "@metamask/announcement-controller": "^4.0.0",
     "@metamask/approval-controller": "^3.4.0",
-    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@npm%3A22.0.0#~/.yarn/patches/@metamask-assets-controllers-npm-22.0.0-6af3715fb2.patch",
+    "@metamask/assets-controllers": "23.0.0",
```
Comment (Gudahtt) [inline package.json:252]: Could you use a `^` range here instead? We don't pin to specific versions unless we have to typically

```suggestion
    "@metamask/assets-controllers": "^23.0.0",
```

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Review by Gudahtt — DISMISSED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**